mobile/src/main/java/org/hawkular/client/android/service/AcknowledgeAlertService.java

```diff
@@ -0,0 +1,21 @@
+package org.hawkular.client.android.service;
+
+import org.hawkular.client.android.backend.model.Trigger;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.POST;
+
+/**
+ * Created by pallavi on 26/06/17.
+ */
+
+public interface AcknowledgeAlertService {
```

Comment:
@garg-anuj You are right. I have removed redundant service classes and added functions for them in their respective classes. I have squashed this commit and the one which removes redundant service classes to commit 23a91c6 Have a look.